.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Feature Requests & Questions
+    url: https://github.com/excalidraw/excalidraw/discussions
+    about: Please ask and answer questions here.
```

Comment:
we should also add a link to Discord (questions, discussions, help with integration...)